src/aat/resources/features/F-044/F-044.feature

```diff
@@ -0,0 +1,136 @@
+    And a case that has just been created as in [Standard_Full_Case_Creation_Data]
+    And a successful call [to get an event token for the case just created] as in [F-044-Prerequisite]
+    When a request is prepared with appropriate values
+    And the request [contains a case Id for just created case]
+    And the request [contains a token created as in F-044-Prerequisite]
+    And it is submitted to call the [Submit event creation as Case worker] operation of [CCD Data Store]
+    Then a positive response is received
+    And the response [includes the updated case details, along with a HTTP 201 Created]
+    And the response has all other details as expected
+
+  @S-279
+  Scenario: must return negative response when request does not provide valid authentication credentials
+    Given a user with [an active profile in CCD]
+    When a request is prepared with appropriate values
+    And the request [does not provide valid authentication credentials]
+    And it is submitted to call the [Submit event creation as Case worker] operation of [CCD Data Store]
+    Then a negative response is received
+    And the response [includes a HTTP 403 Forbidden]
+    And the response has all other details as expected
+
+  @S-280
+  Scenario: must return negative response when request does not provide an authorised access
+    Given a user with [an active profile in CCD]
+    When a request is prepared with appropriate values
+    And the request [does not provide authorised access to the operation]
+    And it is submitted to call the [Submit event creation as Case worker] operation of [CCD Data Store]
+    Then a negative response is received
+    And the response [includes a HTTP 403 Forbidden]
+    And the response has all other details as expected
+
+  @S-281 @Ignore # This scenario is returning 400 instead of expected 404, linked to defect JIRA-6868
+  Scenario: must return negative response when request contains a non-existing case reference
+    Given a user with [an active profile in CCD]
+    When a request is prepared with appropriate values
+    And the request [contains a non-existing case reference]
+    And it is submitted to call the [Submit event creation as Case worker] operation of [CCD Data Store]
+    Then a negative response is received
+    And the response [includes a HTTP 404 'Bad Request']
+    And the response has all other details as expected
+
+  @S-282
+  Scenario: must return 409 when case is altered out of the transaction
+    Given a user with [an active profile in CCD]
+    And a case that has just been created as in [Standard_Full_Case_Creation_Data]
+    And a successful call [to get an event token for the case just created] as in [F-044-Prerequisite]
+    And another successful call [to update that case with the token just created] as in [F-044-Prerequisite_CaseUpdate]
+    When a request is prepared with appropriate values
+    And the request [contains a case Id for just updated case]
```

Comment:
Please update : 
contains a case that has just been created as in Standard_Full_Case_Creation_Data